src/Frontend/Asset/LocaleJsCompiler.php

```diff
@@ -22,23 +22,12 @@ public function setTranslations(array $translations)
 
     public function compile()
     {
-        $output = "
- System.register('locale', [], function(_export) {
-    return {
-        execute: function() {
-            _export('default', function(app) {
-                app.translator.translations = ".json_encode($this->translations).";\n";
+        $output = 'flarum.core.app.translator.translations='.json_encode($this->translations).";\n";
```

Comment:
@tobscure Please review whether we need some JSON encoding options here, for security. (Compare with Laravel's `CompilesJson.php`, for example.)

Reply:
Good thought. Since this is being outputted in a JavaScript file (not within HTML), no encoding options are required. https://stackoverflow.com/a/6817147/887933

Reply:
Ah, cheers for double checking.